### ISSUE TITLE
Sample end time drifted

### ISSUE THREAD

**Issue by raacampbell** [SainsburyWellcomeCentre/StitchIt#140]
This happened:
```
AMS_AF_SST2_rab (160/160) — 50.0 µm cuts — (2019/06/19 08:24:18) 
laser power: 35%; averaging: 4 frames - Chan 2 MONTAGE; Finishing at 12:21 on Wed 19/06 
```
But it finished at 08:30. Acq end time correctly said 08:30 at the start then progressively drifted and it never said "FINISHING SOON".

**Comment by raacampbell** [comment]
May be related to the changes made recently to the prettyTime scripts? But that would be odd, since it's supposed to get these times from the acquisition log file... Unless `prettyTime` in BakingTray is writing those numbers and is wrong?

**Comment by raacampbell** [comment]
Fixed: 3efd6520f85d8cccd0f030b8fe0ab623440ca1bb
Some completion times lacked the seconds because because `0 secs` was not being printed to the acq log file. Changed the regex and now it's OK.
